src/components/OrgPostCard/OrgPostCard.spec.tsx

```diff
@@ -175,6 +178,72 @@ vi.mock('utils/errorHandler', () => ({
   errorHandler: vi.fn(),
 }));
 
+// Mock useParams hook
+vi.mock('react-router-dom', async () => {
+  const actual = await vi.importActual('react-router-dom');
+  return {
+    ...actual,
+    useParams: vi.fn().mockReturnValue({ orgId: 'test-org-id' }),
+  };
+});
+
+// Mock MinioUpload hook
+vi.mock('utils/MinioUpload', () => ({
+  useMinioUpload: () => ({
+    uploadFileToMinio: vi.fn().mockResolvedValue({
+      objectName: 'mocked-object-name',
+      fileHash: 'mocked-file-hash',
+    }),
+  }),
+}));
+
+// Add mocks for presigned URLs
+const presignedUrlImageMock = {
+  request: {
+    query: PRESIGNED_URL,
+    variables: {
+      input: {
+        objectName: 'test-image-object-name',
+        operation: 'GET',
+      },
+    },
+  },
+  result: {
+    data: {
+      createPresignedUrl: {
+        fileUrl: 'https://minio-server.example/test-image.jpg',
```

Comment:
1. The develop-postgres branch of the API no longer uses `fileurl` therefore your mock is incorrect. Using `fileurl` in the app will cause it to malfunction.
2. You need to use `objectName` instead 
3. You also cannot use `objectName` in the preview. This is impossible. You have to use the  browser's URL object method to preview the images and the MinioUpload.ts helper function for uploading images.